### PR TITLE
Mocks

### PR DESCRIPTION
**Pull request by rborn**
Fix for #8 

@shichongrui is this something that you thing it's good to be like this?
I tested it and it works pretty nice, maybe we need to identify what other components need to be mocked, but for a start I think it should be fine

let me know 🤗

### CODE REVIEW

**Comment by DaKaZ** [comment]
@rborn looks good to me! I manually mock this stuff all the time so these addtions would be great.  What do you think @shichongrui can we get this PR in?

File: package.json [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
   "devDependencies": {
-    "babel-jest": "^20.0.3",
-    "babel-preset-react-native": "^2.1.0",
-    "jest": "^20.0.4",
-    "react": "16.0.0-alpha.12",
-    "react-native": "^0.46.1",
-    "react-test-renderer": "16.0.0-alpha.12"
+    "babel-jest": "^23.0.0",
+    "babel-preset-react-native": "4.0.0",
+    "jest": "*",
```
Comment (shichongrui) [inline package.json]: Curious why these need to change? Setting dependencies to * can be dangerous.

**Review by shichongrui — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rborn) [reply package.json]: Donno, I must have played with something and forgot to revert 🤦‍♂️. I'll fix it this week and update the PR

**Comment by rborn** [comment]
@shichongrui @DaKaZ should be fixed now, sorry for the delay 🤗

**Comment by DaKaZ** [comment]
@shichongrui can we get this merged??

**Comment by shouse** [comment]
@shichongrui Any update with this?  It would be very useful for a lot of tests that I have, and would make my code a lot more DRY.  

BTW, thanks for this testing lib!

**Review by shichongrui — APPROVED** [review]

**Comment by shichongrui** [comment]
Published to npm as version 1.3.0
https://www.npmjs.com/package/react-native-test-utils

**Pull request merged**